x-pack/plugins/canvas/server/routes/workpad/workpad_schema.ts

```diff
@@ -97,5 +97,5 @@ export const WorkpadSchema = schema.object({
   page: schema.number(),
   pages: schema.arrayOf(WorkpadPageSchema),
   width: schema.number(),
-  theme: schema.maybe(WorkpadThemeSchema),
+  theme: schema.maybe(schema.nullable(WorkpadThemeSchema)),
```

Comment:
This is to allow for the creation of a new workpad (It was failing on validation previously).